### PR TITLE
Feature / wannagolist

### PR DESCRIPTION
**Pull request by gergana95**
Past events counting and rendering in WishList component.
Added wishListed prop to Cards and changed the logic in Attend component.
FetchWishListIfNeeded - handling Api wishlist fetching

### CODE REVIEW

File: src/Events/WishList/index.js [hunk 1]
```diff
@@ -66,22 +72,57 @@ class WishList extends Component {
...
+     */
     countEventsByMonth = (_events) => {
         let monthEventsNumber;
-        _.forEach(_events, (group, key) => {
+        _.forEach(_events, (group) => {
             _.forEach(group, (e) => {
-                monthEventsNumber= +_.size(e);
+                monthEventsNumber = +_.size(e);
             });
         });
```
Comment (Bratanov) [inline src/Events/WishList/index.js]: You can try:
```
for(group of _events) {
    for(event of group) {
        ..
    }
}
```
Lodash has great helpers, but for-of can be very useful 🦃

Comment (Bratanov) [reply src/Events/WishList/index.js]: As discussed with @gergana95, for-of works great for arrays, but `_events` is an object. for-in is a little ugly in this case.
I'll try to make my next comment wiser.

File: src/common/Card.js [hunk 2]
```diff
@@ -31,9 +32,12 @@ class Card extends Component {
         return { __html: this.props.event.shortDescription };
     }
 
-    renderTags() {
+    renderTags(_tags) {
+        if (_tags === undefined) {
```
Comment (superKalo) [inline src/common/Card.js]: typeof _tags === 'undefined'

File: src/service.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import moment from 'moment';
+
+export const orderEventsByMonth = (_wishList) => {
```
Comment (superKalo) [inline src/service.js]: Please add a comment 🌈

**Pull request merged**